styles.css

```diff
@@ -65,12 +78,50 @@ body {
   font-size: 18px;
   line-height: 19px;
   text-decoration: none;
+  padding-left: 12px;
 }
 
 .nav_icons {
   display: none;
 }
 
+/* Mobile Menu */
+
+.mb_menu {
+  background-color: #3c3a39;
+  position: absolute;
+  top: 0;
+
+  /* left: 100%; */
+  display: none;
+  padding-right: 1px;
+}
```

Comment:
- [ ] Kindly make the mobile menu `position: fixed;` so it stays at its position even if you scroll.
- [ ] Also give it `width: 100vw;` and `height: 100vh;` so it takes the whole width and height of the screen.

Your Design | Figma Design
------------- | ----------------
![Capture](https://user-images.githubusercontent.com/33752735/194944977-329cd78f-3384-4e05-bbf6-e1b7de4d95f0.PNG) | ![Capture2](https://user-images.githubusercontent.com/33752735/194945002-9a4c7d54-2fcf-465c-af87-a46591a7a228.PNG)